### PR TITLE
etcdctl: "fields" output formats

### PR DESCRIPTION
**Pull request by heyitsanthony**
Writes out fields from responses in the format "FieldName" : FieldValue. If FieldValue is a string, it is formatted with %q.

### CODE REVIEW

File: etcdctl/ctlv3/command/printer.go [hunk 1]
```diff
@@ -329,3 +331,127 @@ func printPB(m pbMarshal) {
...
+	for _, k := range r.Keys {
+		fmt.Printf("\"Key\" : %q\n", string(k))
+	}
+}
+
+func (p *fieldsPrinter) MemberList(r v3.MemberListResponse) {
+	p.hdr(r.Header)
+	for _, m := range r.Members {
+		fmt.Println(`"ID" :`, m.ID)
+		fmt.Println("\"Name\" : %q\n", m.Name)
```
Comment (gyuho) [inline etcdctl/ctlv3/command/printer.go]: `fmt.Printf`?

**Comment by gyuho** [comment]
lgtm. /cc @xiang90

**Comment by xiang90** [comment]
Can we add documentation about this output format?

**Comment by heyitsanthony** [comment]
@xiang90 yes, but I'm noticing the documentation is very bad for formatting. A lot of commands are documented as returning simple replies even though the format option is being ignored; for some cases formatting doesn't make much sense, for others it does but was never implemented. This may need a few passes.

File: etcdctl/ctlv3/command/printer.go [hunk 2]
```diff
@@ -329,3 +331,127 @@ func printPB(m pbMarshal) {
...
+	fmt.Printf("\"%sKey\" : %q\n", pfx, string(kv.Key))
+	fmt.Printf("\"%sCreateRevision\" : %d\n", pfx, kv.CreateRevision)
+	fmt.Printf("\"%sModRevision\" : %d\n", pfx, kv.ModRevision)
+	fmt.Printf("\"%sVersion\" : %d\n", pfx, kv.Version)
+	fmt.Printf("\"%sValue\" : %q\n", pfx, string(kv.Value))
+	fmt.Printf("\"%sLease\" : %d\n", pfx, string(kv.Lease))
+}
+
+func (p *fieldsPrinter) hdr(h *pb.ResponseHeader) {
+	fmt.Println(`"ClusterId" :`, h.ClusterId)
```
Comment (xiang90) [inline etcdctl/ctlv3/command/printer.go]: ClusterID

File: etcdctl/ctlv3/command/printer.go [hunk 3]
```diff
@@ -329,3 +331,127 @@ func printPB(m pbMarshal) {
...
+	fmt.Println(`"MemberId" :`, h.MemberId)
```
Comment (xiang90) [inline etcdctl/ctlv3/command/printer.go]: MemberID

**Comment by xiang90** [comment]
LGTM

**Pull request merged**